mava/components/jax/executing/base.py

```diff
@@ -46,7 +47,18 @@ def on_execution_init_start(self, executor: SystemExecutor) -> None:
         Returns:
             None.
         """
-        executor._interval = self.config.interval  # type: ignore
+
+        if self.config.evaluation_duration != 1 and (
```

Comment:
could we also raise an error if evaluator duration is  > evaluator interval (for episodes key)

Reply:
I am not sure whether we should? I think we run the risk of having to cater for every possible configuration that could lead to weird evaluation. The reason we raise this in particular is because passing in an `evaluation_duration` parameter without giving an `evaluation_interval` will lead to the duration being ignored by the system. But if you think we should I am happy to add another error in?

Reply:
Makes sense. We discussed this offline. The main thing is that the code must not crash which it was explained, will not happen